lib/blather/client/client.rb

```diff
@@ -277,7 +277,7 @@ def ready!
 
     def client_post_init
       write_with_handler Stanza::Iq::Roster.new do |node|
-        roster.process node
+        roster.process(node) unless node.error?
```

Comment:
I could imagine marking the roster as unavailable being useful, it just being empty on the other hand seems ok, too. What do you think?